tests/subsys/fs/fat_fs_api/src/test_fat.h

```diff
@@ -9,7 +9,17 @@
 #include <zephyr/ztest.h>
 #include <zephyr/fs/fs.h>
 
-#define FATFS_MNTP	"/NAND:"
+#ifdef CONFIG_DISK_DRIVER_RAM
+#define DISK_NAME CONFIG_DISK_RAM_VOLUME_NAME
+#elif defined(CONFIG_DISK_DRIVER_FLASH)
+#define DISK_NAME DT_PROP(DT_NODELABEL(test_disk), disk_name)
+#elif defined(CONFIG_DISK_DRIVER_SDMMC)
+#define DISK_NAME SDMMC_VOLUME_NAME
+#else
```

Comment:
please add support for MMC in this PR (relevant kconfigs here: CONFIG_DISK_DRIVER_MMC and MMC_VOLUME_NAME)

Reply:
Done.
Also fixed the line 17, by adding missing `CONFIG_`